app/middlewares/checkAuthentication.js

```diff
@@ -47,7 +47,7 @@ module.exports.validate = async (req, res, next) => {
 module.exports.validateAdmin = async (req, res, next) => {
   try {
     const decodedToken = await decodeToken(req.headers.token);
```

Comment:
Not gonna reject you for this, but you changed the const to `decodeResult ` in `validate` but left it as is in `validateAdmin` 😝
More importantly, merge your changes so I can approve.

Reply:
aksjkasjkaksksa i ll change it